app/routes.py

```diff
@@ -0,0 +1,357 @@
+from app import db
+from flask import Blueprint, jsonify, request
+from datetime import datetime, timedelta
+from app.models.video import Video
+from app.models.customer import Customer
+from app.models.rental import Rental
+
+
+videos_bp = Blueprint("videos", __name__, url_prefix=("/videos"))
+
+#POST/CREATE 
+@videos_bp.route("",methods=["POST"])
```

Comment:
👍 

Nice job splitting your endpoints into separate functions. There are still shared pieces of functionality in some of the endpoints that we can think about trying to reduce duplication (especially looking up a record by its id), but on the plus side, in each function, we only need to think about the logic related to that specific endpoint.